tools/mo/unit_tests/mo/moc_frontend/moc_extractor_test_actual.py

```diff
@@ -85,7 +84,7 @@ def test_decode_name_with_port_delim_op_out(self):
         place_stat = get_place_statistic()
 
         assert model_stat.get_place_by_tensor_name == 1
-        assert model_stat.get_place_by_operation_name == 1
+        assert model_stat.get_place_by_operation_name == 2
```

Comment:
I checked - 'get_place_by_operation_name' for `operation` is called twice from MO now which looks a bit redundant. Originally test was intended to verify that there is no redundant calls to frontend are happened from MO side.
If it is not possible to avoid redundant calls now, let's set just `assert model_stat.get_place_by_operation_name > 1`

Reply:
There is no way to reduce the number of function calls to 1 in every possible scenario, but I was able to achieve it for the majority of the available combinations of node names